sormas-rest/src/main/java/de/symeda/sormas/rest/security/KeycloakHttpAuthenticationMechanism.java

```diff
@@ -54,6 +55,10 @@ public AuthenticationStatus validateRequest(HttpServletRequest request, HttpServ
 		Credential credential = new CallerOnlyCredential(keycloakContext.getToken().getPreferredUsername());
 		IdentityStoreHandler identityStoreHandler = CDI.current().select(IdentityStoreHandler.class, new Annotation[0]).get();
 		CredentialValidationResult result = identityStoreHandler.validate(credential);
+		if (result.getStatus() == CredentialValidationResult.Status.VALID) {
```

Comment:
Seems to be a hack, but I know it's hard to find the right way to force the usage of roles returned by the identity store.

Reply:
Unfortunately the whole way we use Keycloak with SORMAS feels like a hack 🙃

Reply:
I'm somewhat concerned that we don not do any authentication at all, at least I cannot see where we check the actual token from keycloak. 

`hasValidAuthorization` does only check the header being present. Then we use `CallerOnlyCredential` which basically seems to ignore the token altogether and then `KeycloakIdentityStore.validate` consequently does not care about the token at all. Can you point me to the code where the Java code checks either a cookie or a token being correctly issued by the registered Keyclaok?